data_tfms.ipynb

```diff
@@ -39,7 +39,7 @@
   },
```

Comment:
<p><code>trail.text</code> is missing from data folder</p><pre class="ql-syntax" spellcheck="false">trial_df = parse_file("data/raw/trial.txt"); len(trial_df)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Which file was this? Do we have a copy of this somewhere?</p>

Reply:
Downloaded by `download.ipynb`

Reply:
file added by 4ec581d

Reply:
Resolved by 6885478